tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/css/styles.css

```diff
@@ -1799,4 +1799,21 @@ div.scrollbar > pre::-webkit-scrollbar-thumb {
     from(#555),
     to(#555),
     color-stop(.6, #555))
-}
\ No newline at end of file
+}
+
+.isDisabled {
+    cursor: not-allowed;
+    opacity: 0.5;
+    color: #02aaad;
+}
+
+.isDisabled {
+    color: #02aaad;
+    display: inline-block;
```

Comment:
Why we have 2 tags for isDisabled?

Reply:
13712b2ee95c1b9ae9345ac8be85cefbbe170fd9 this commit fix above requirement